src/openforms/prefill/contrib/objects_api/api/serializers.py

```diff
@@ -15,3 +21,50 @@ class PrefillTargetPathsSerializer(serializers.Serializer):
         label=_("json schema"),
         help_text=_("Corresponding (sub) JSON Schema of the target path."),
     )
+
+
+class ObjecttypeVariableMappingSerializer(serializers.Serializer):
+    """A mapping between a form variable key and the corresponding Objecttype attribute."""
+
+    variable_key = FormioVariableKeyField(
+        label=_("variable key"),
+        help_text=_(
+            "The 'dotted' path to a form variable key. The format should comply to how Formio handles nested component keys."
+        ),
+    )
+    target_path = serializers.ListField(
+        child=serializers.CharField(label=_("Segment of a JSON path")),
+        label=_("target path"),
+        help_text=_(
+            "Representation of the JSON target location as a list of string segments."
+        ),
+    )
+
+
+class ObjectsAPIOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
+    objects_api_group = PrimaryKeyRelatedAsChoicesField(
+        queryset=ObjectsAPIGroupConfig.objects.exclude(
+            Q(objects_service=None)
+            | Q(objecttypes_service=None)
+            | Q(drc_service=None)
+            | Q(catalogi_service=None)
```

Comment:
I don't think that Documents API/Catalogi API are used at all in the prefill layer, so these filters can be removed. We only need a working Object Types API and Objects API to perform the actual lookup at some point.